test/testdata/generate.sh

```diff
@@ -109,7 +109,7 @@ function scaffold_test_project {
     $kb create api --group crew --version v1 --kind Admiral --controller=true --resource=true --namespaced=false --make=false
   elif [[ $project == "project-v3-with-deploy-image" ]]; then
       header_text 'Creating Memcached API with deploy-image plugin ...'
-      $kb create api --group example.com --version v1alpha1 --kind Memcached --image=memcached:1.4.36-alpine --image-container-command="memcached,-m=64,-o,modern,-v" --image-container-port="11211" --plugins="deploy-image/v1-alpha" --make=false --namespaced=false
+      $kb create api --group example.com --version v1alpha1 --kind Memcached --image=memcached:1.4.36-alpine --image-container-command="memcached,-m=64,-o,modern,-v" --image-container-port="11211" --run-as-user="1001" --plugins="deploy-image/v1-alpha" --make=false --namespaced=false
```

Comment:
HI @NikhilSharmaWe :
Could we add here a scaffold that does not pass the flags so we also are able to verify/review it easily?
```suggestion
      $kb create api --group example.com --version v1alpha1 --kind Memcached --image=memcached:1.4.36-alpine --image-container-command="memcached,-m=64,-o,modern,-v" --image-container-port="11211" --run-as-user="1001" --plugins="deploy-image/v1-alpha" --make=false --namespaced=false
      $kb create api --group example.com --version v1alpha1 --kind Busybox --image=busybox:1.28 --plugins="deploy-image/v1-alpha" --make=false --namespaced=false
```

Reply:
Also, it helps us ensure/test in the ci that the plugin can properly work with more than one API scaffold.